lib/keys.js

```diff
@@ -41,7 +41,20 @@ Keys.prototype.create = function(fn) {
 
     data[key] = true;
     keys.writeFile(data, function(err) {
-      fn(err, key);
+      if(err){
+        if(err.code == 'ENOENT' && err.path == '.dpd/keys.json'){
+          console.error('Path: '+err.path+' not found, creating new one.');
+          // TODO: Could ask with prompt?
+          fs.mkdirSync('.dpd');
+          keys.writeFile(data,function(error){
```

Comment:
Shall we ensure folder before write?  instead of write > err > create folder > write again ? 

By the way, it seems fix a bug. Just want to know how can we hit this bug?